### PR TITLE
@jorystiefel => [Search] Do not allow rotation on iPhone.

### LINKED ISSUE #855: search supports rotation
open search turn phone, everything turns with you

### PR DESCRIPTION
**Pull request by alloy**
Fixes #855.

### CODE REVIEW

File: Artsy/App/ARAppActivityContinuationDelegate.m [hunk 1]
```diff
@@ -28,10 +29,12 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserAct
...
-    UIViewController *viewController = [ARSwitchBoard.sharedInstance loadURL:URL];
-    if (viewController) {
-        [[ARTopMenuViewController sharedController] pushViewController:viewController];
-    }
+    [ArtsyAPI getXappTokenWithCompletion:^(NSString *xappToken, NSDate *expirationDate) {
+        UIViewController *viewController = [ARSwitchBoard.sharedInstance loadURL:URL];
+        if (viewController) {
+            [[ARTopMenuViewController sharedController] pushViewController:viewController];
+        }
+    }];
```
Comment (alloy) [inline Artsy/App/ARAppActivityContinuationDelegate.m]: Doh, did not actually stash :D

**Comment by orta** [comment]
:+1:

**Comment by jorystiefel** [comment]
🌀No rotation! :+1:

**Comment by alloy** [comment]
I wonder if we should even really do it this way on each individual VC instead of once on e.g. the top menu VC.

**Comment by orta** [comment]
Definitely value in doing that, in iPhone there needs to be some exceptions around the ArtworksVC but that's basically it.

**Pull request merged**